pythran/pythonic/types/NoneType.hpp

```diff
@@ -340,6 +316,23 @@ namespace pythonic
   }
 
   template <class T>
+  bool from_python<types::none<T>>::is_convertible(PyObject *obj)
```

Comment:
why do you need this? If I remember well, you either have a None, and then it's a none_type, or an object and then it's a plain object?